src/content/docs/upgrade-install-new-operator.md

```diff
@@ -0,0 +1,151 @@
+title: Upgrade Installation(v2.0,2.1->v2.2)
+permalink: /docs/getting-started/upgrade-install-new-operator
+style_class: quick-installation
+---
+## disclaimer
+
+Version 2.2 represents an alpha release, exclusively accessible via the "fast" channel.
+Subsequent releases will transition to the "fast" channel once the new operator attains greater stability.
+
+For installation steps of the old (version 1.X, stable),
+see [quick installation of the 1.X version](../quick-installation).
+
+For installation steps of the old (version 2.0, 2.1, fast),
+see [quick installation of the 2.X version](../quick-installation-new-operator).
+
+### Pre-requisites
+
+Below information is only appliable for Open Data Hub Operator v2.0.0 and forth release.
+
+Installing Open Data Hub requires OpenShift Container Platform version 4.10+.
+All screenshots and instructions are from OpenShift 4.12.
+For the purposes of this quick start, we used [try.openshift.com](https://try.openshift.com/) on AWS.
+
+Tutorials will require an OpenShift cluster with a minimum of 16 CPUS and 32GB of memory across all OpenShift worker nodes.
+
+### Installing v2.2 Open Data Hub Operator
+
+The Open Data Hub operator is available for deployment in the OpenShift OperatorHub as a Community Operators. You can install it from the OpenShift web console by following the steps below:
+
+1. From the OpenShift web console, log in as a user with `cluster-admin` privileges.  For a developer installation from [try.openshift.com](https://try.openshift.com/), the `kubeadmin` user will work.
+   ![Log in to OpenShift](../assets/img/pages/docs/quick-installation/login.png)
+
+2. On the lefthand bar, from `Operators` -> `OperatorHub`,
+   - filter for `Open Data Hub Operator`.
+   - select `AI/Machine Learning` and look for the icon for `Open Data Hub Operator`.
+   ![OperatorHub](../assets/img/pages/docs/quick-installation-new-operator/operator-hub.png "OperatorHub")
+
+3. Click `Continue` in the "Show community Operator" dialog if it pops out. Click the `Install` button to install the Open Data Hub operator.
+   ![Install](../assets/img/pages/docs/quick-installation-new-operator/installation1.png "Install")
+
+4. The subscription creation view will offer a few options including `Update Channel`, make sure the `fast` channel is selected. Click `Install` to deploy the opendatahub operator into the `openshift-operators` namespace.
+   ![Select Channel](../assets/img/pages/docs/quick-installation-new-operator/channels.png "Select Channel")
+
+5. To view the status of the Open Data Hub operator installation, find the Open Data Hub Operator under `Operators` -> `Installed Operators`. It might take a couple of minutes to show, but once the `Status` field displays `Succeeded`, you can proceed to create a DataScienceCluster instance
+   ![Installed Operators](../assets/img/pages/docs/quick-installation-new-operator/operator-installed.png "Installed Operators")
+
+### Create a DataScienceCluster instance
```

Comment:
Before this can we add a section called

`[Optional] Configure applications Namespace` and just add a step to access DSCInitialization with a screenshot of configured namespace

Reply:
done